ts/sagas/__tests__/wallet.test.ts

```diff
@@ -133,3 +134,47 @@ describe("startOrResumeAddCreditCardSaga", () => {
       .next();
   });
 });
+
+describe("deleteUnsuccessfulActivePaymentSaga", () => {
+  describe("when there is no outcomeCode", () => {
+    it("it should do nothing", () => {
+      testSaga(testableWalletsSaga!.deleteUnsuccessfulActivePaymentSaga)
+        .next()
+        .select(lastPaymentOutcomeCodeSelector)
+        .next({ outcomeCode: none })
+        .isDone();
+    });
+  });
+
+  describe("when there is an outcomeCode and it is not a success", () => {
+    it("it should put runDeleteActivePaymentSaga", () => {
+      testSaga(testableWalletsSaga!.deleteUnsuccessfulActivePaymentSaga)
+        .next()
+        .select(lastPaymentOutcomeCodeSelector)
+        .next({ outcomeCode: some({ status: "errorBlocking" }) })
+        .put(runDeleteActivePaymentSaga())
+        .next()
+        .isDone();
+
+      testSaga(testableWalletsSaga!.deleteUnsuccessfulActivePaymentSaga)
+        .next()
+        .select(lastPaymentOutcomeCodeSelector)
+        .next({ outcomeCode: some({ status: "errorTryAgain" }) })
+        .put(runDeleteActivePaymentSaga())
+        .next()
+        .isDone();
+    });
+  });
+
+  describe("when there is an outcomeCode and it is a success", () => {
+    it("it should do nothing", () => {
+      testSaga(testableWalletsSaga!.deleteUnsuccessfulActivePaymentSaga)
+        .next()
+        .select(lastPaymentOutcomeCodeSelector)
+        .next({ outcomeCode: some({ status: "success" }) })
+
+        .next()
```

Comment:
```suggestion
```

 This `.next()` here will make the test pass even with a status different from `"success"`, am I missing something or it could be removed? 😀

Reply:
ops! ty!
f0a9fd466b7b1fad1cd6810ec2f3b8c649ea6f9b